pkgs/tools/filesystems/ceph/default.nix

```
@@ -498,6 +498,11 @@ rec {
         --replace "/sbin/modprobe" "${kmod}/bin/modprobe" \
         --replace "/bin/grep" "${gnugrep}/bin/grep"
 
+      # Patch remount to use full path to mount(8), otherwise ceph-fuse fails when run
+      # from a systemd unit for example.
+      substituteInPlace src/client/fuse_ll.cc \
+        --replace "mount -i -o remount" "${util-linux}/bin/mount -i -o remount"
```

```suggestion
        --replace-fail "mount -i -o remount" "${util-linux}/bin/mount -i -o remount"
```
yep, the other shoudl use --replace-fail, too

---

Fixed in my tree, I'll force-push an updated PR once I've built and tested it (Ceph build takes ages...).